advocacy_docs/kubernetes/cloud_native_postgresql/index.mdx

```diff
@@ -74,7 +74,7 @@ and is available under the [EnterpriseDB Limited Use License](https://www.enterp
 You can [evaluate Cloud Native PostgreSQL for free](evaluation.md).
 You need a valid license key to use Cloud Native PostgreSQL in production.
 
-!!! Important
+!!! Note
```

Comment:
At a glance this looked like a scary warning, when it's actually advising of a feature. I'm not sure if `Note` is the right header here but I don't think we want this box to be red.

/cc @gbartolini